src/test/java/org/example/c03/controlFlow/DiagonalStarTest.java

```diff
@@ -0,0 +1,20 @@
+package org.example.c03.controlFlow;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DiagonalStarTest {
+    @Test
+    public void generateRectangle() {
```

Comment:
Use SqareStar instead of Rectangle in test name

Reply:
done